packages/amplify-nodejs-template-provider/amplify-plugin.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "functionTemplate",
+  "id": "f579f2fa-35c1-49c4-950f-b588e779b4dd",
```

Comment:
this is tied to the nodejs provider right? Will this affect the existing platform?

Reply:
No, it's not used anymore. It won't break anything with or without it.